web/tests/calculate-percentage-directive-spec.js

```diff
@@ -43,13 +43,21 @@ describe('calculatePercentage directive', function() {
             this.elm = this.$compile(tpl)(this.scope);
             this.scope.$digest();
         };
+        spyOn(this.calculateCompletionPercent, 'countQuestionsTotal').and.returnValue(questionSet);
+        spyOn(this.calculateCompletionPercent, 'countNumberOfAnsweredQuestions').and.returnValue(answers);
     });
 
     it('should call this.calculateCompletionPercent.countQuestionsTotal with the questionSet', function() {
-        spyOn(this.calculateCompletionPercent, 'countQuestionsTotal').and.returnValue(questionSet);
+        this.scope.field = questionSet;
```

Comment:
Maybe the problem is that you have `field` here but in the template you have `fields` ?